roles/ovirt-vm-infra/README.md

```diff
@@ -40,6 +40,12 @@ The `profile` dictionary can contain following attributes:
 | domain             | UNDEF                 | The domain of the virtual machine.                         |
 | root_password      | UNDEF                 | The root password of the virtual machine.                      |
 | high_availability  | UNDEF                 | Whether or not the node should be set highly available. |
+| storage_domain     | UNDEF                 | Name of the storage domain where all template disks should be created. |
```

Comment:
s/Name of the storage domain where all template disks should be created./Name of the storage domain where all virtual machine disks should be created.